src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -82,17 +82,17 @@ public enum MethodSemanticsAttributes
 }
 namespace System.Reflection.Metadata
```

Comment:
@tmat @nguerrera could you guys please have a scan over the changes to this ref?

Reply:
I took a look and it looks fine.